uber/site_sections/jobs.py

```diff
@@ -83,14 +85,23 @@ def signups(self, session, department_id=None, message=''):
             'checklist': department_id and session.checklist_status('postcon_hours', department_id)
         }
 
-    def everywhere(self, session, message='', show_restricted=''):
+    def everywhere(self, session, message='', show_restricted='', show_nonpublic=''):
+        job_filters = [Job.start_time > localized_now() - timedelta(hours=2)]
+        if show_restricted:
```

Comment:
Looks like this should be `if not show_restricted` -- per your comment, I'll fix it now.